pkg/ui/react-app/src/pages/graph/Panel.tsx

```diff
@@ -429,23 +514,42 @@ class Panel extends Component<PanelProps & PathPrefixProps, PanelState> {
                 <option value="thanos">Thanos</option>
               </Input>
             </div>
-            <div className="float-right">
+            <div className="float-right" onMouseEnter={this.handleMouseEnter} onMouseLeave={this.handleMouseLeave}>
               <Checkbox
                 wrapperStyles={{ marginRight: 20, display: 'inline-block' }}
-                id={`explain-${id}`}
-                onChange={this.handleChangeExplain}
-                checked={options.explain}
-                disabled={options.disableExplainCheckbox}
+                id={`analyze-${this.props.id}`}
+                onChange={this.handleChangeAnalyze}
+                checked={options.analyze}
+                disabled={options.disableAnalyzeCheckbox}
+                className="analyze-checkbox"
               >
-                Explain
+                Analyze
               </Checkbox>
+              {/* <Tooltip
```

Comment:
Why is this commented out?

Reply:
was just testing where the UI tests are failing, will uncomment it

Reply:
added customized tooltip instead of the component, the UI tests were failing due to "target" attribute